orderer/common/follower/follower_chain.go

```diff
@@ -20,7 +24,12 @@ import (
 
 //TODO skeleton
 
-//go:generate mockery -dir . -name LedgerResources -case underscore -output mocks
+const (
+	defaultPullRetryMinInterval time.Duration = 10 * time.Millisecond
+	defaultPullRetryMaxInterval time.Duration = 30 * time.Second
```

Comment:
I'd also think perhaps 30 seconds is a little short for a max? I'm less concerned here than the 10ms above, but maybe extend this to be 60s?

Reply:
Done